code/processes/gateway.q

```diff
@@ -388,6 +388,10 @@ getserverscross:{[req;att;besteffort]
 getserverids:{[att]
   if[99h<>type att;
    // its a list of servertypes e.g. `rdb`hdb
+   // check if user attributes are a symbol list
+   if[not type[att]in 99 11h;
```

Comment:
@claire-sloan is this how you wanted these combined? Second check for 99h is redundant in this scenario.

Reply:
The function still needs a way to differentiate between `att` being a list of symbols and it being a dictionary so I think we may need to keep them separate.

Reply:
Yeah, I think this check should just be `if[not type[att]=11h;`